pkgs/development/python-modules/skorch/default.nix

```diff
@@ -96,5 +96,10 @@ buildPythonPackage rec {
     changelog = "https://github.com/skorch-dev/skorch/blob/master/CHANGES.md";
     license = lib.licenses.bsd3;
     maintainers = with lib.maintainers; [ bcdarwin ];
+    badPlatforms = [
+      # Most tests fail with:
+      # skorch/tests/callbacks/test_lr_scheduler.py
```

Comment:
I don't exactly understand the comment - most tests in this file fail, or what?

Reply:
This was a mistake when I copied from my terminal.
Fixed.